sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AppendBlobAPITest.groovy

```diff
@@ -494,4 +494,11 @@ class AppendBlobAPITest extends APISpec {
         null                  | null                    | garbageEtag   | null
         null                  | null                    | null          | receivedEtag
     }
+
+    def "Get Append Blob Name"() {
+        given:
+        def appendBlobName = "test_name"
+        expect:
+        appendBlobName == bc.getName()
```

Comment:
How are we ensuring the client is using the name `test_name`

Reply:
Wrong tests. Rewrote